package.json

```diff
@@ -24,11 +24,13 @@
   "dependencies": {
     "node-uuid": "1.4.x",
     "lru-cache": "~2.2.1",
-    "json-stringify-safe": "~5.0.0"
+    "json-stringify-safe": "~5.0.0",
+    "async": "~0.9.0"
   },
   "devDependencies": {
     "express": "*",
     "jade": "~0.27.7",
+    "jslint": "^0.9.0-pre006",
```

Comment:
@coryvirok seems that the problem for the tests not passing is this.

Can we remove it? Or is needed?